app/server/api/auth_api.py

```diff
@@ -2,134 +2,21 @@
 from flask.views import MethodView
 from server import db, sentry
 # from server import limiter
-from server.constants import DENOMINATION_DICT
-from server.models.transfer_account import TransferAccount
 from phonenumbers.phonenumberutil import NumberParseException
 from server.models.user import User
 from server.models.organisation import Organisation
 from server.models.email_whitelist import EmailWhitelist
-from server.models.currency_conversion import CurrencyConversion
 from server.models.blacklist_token import BlacklistToken
-from server.models.transfer_usage import TransferUsage
-from server.utils.intercom import create_intercom_secret
-from server.utils.auth import requires_auth, tfa_logic, show_all
+from server.utils.auth import requires_auth, tfa_logic, show_all, create_user_response_object
 from server.utils.access_control import AccessControl
 from server.utils import user as UserUtils
 from server.utils.phone import proccess_phone_number
-from server.utils.feedback import request_feedback_questions
 from server.utils.amazon_ses import send_reset_email, send_activation_email, send_invite_email
-from server.utils.blockchain_transaction import get_usd_to_satoshi_rate
 
-import time, random
+import random
 
 auth_blueprint = Blueprint('auth', __name__)
 
-
```

Comment:
was kinda gross having this at the top of the api file, so shifted it into utils to be more consistent with other endpoints